test/osm_files/SFD_Successful_EnergyPlus_Run_TMY.osm

```diff
@@ -1,41 +1,41 @@
 !- NOTE: Auto-generated from /test/osw_files/SFD_Successful_EnergyPlus_Run_TMY.osw
 
 OS:Version,
-  {10665648-1be1-4ec2-88e0-705af54af60a}, !- Handle
-  2.9.0;                                  !- Version Identifier
+  {7885ff8e-a76f-4d61-96de-39641d95785c}, !- Handle
+  2.9.1;                                  !- Version Identifier
 
 OS:SimulationControl,
-  {f55ebc1a-69c4-4ab9-935a-e6e3c79bd4cf}, !- Handle
+  {5b572371-7b12-4aa9-bb11-b4b9bfca044a}, !- Handle
   ,                                       !- Do Zone Sizing Calculation
   ,                                       !- Do System Sizing Calculation
   ,                                       !- Do Plant Sizing Calculation
   No;                                     !- Run Simulation for Sizing Periods
 
 OS:Timestep,
-  {686d56a0-d60b-4910-99b2-6c6d17c49a11}, !- Handle
+  {3a2c4897-8235-4fb5-9ac1-15ab64a580eb}, !- Handle
   6;                                      !- Number of Timesteps per Hour
 
 OS:ShadowCalculation,
-  {c27521e2-3e24-4004-b41f-1b66ac93d99e}, !- Handle
+  {8bb19347-cddc-4825-af10-6ca691349070}, !- Handle
   20,                                     !- Calculation Frequency
   200;                                    !- Maximum Figures in Shadow Overlap Calculations
 
 OS:SurfaceConvectionAlgorithm:Outside,
-  {3d972c64-a5e9-4966-9de4-fb7f67c3a1ce}, !- Handle
+  {487854e6-f91d-438a-a5d8-1f2a48756279}, !- Handle
   DOE-2;                                  !- Algorithm
 
 OS:SurfaceConvectionAlgorithm:Inside,
-  {63644801-4383-4e6a-887f-06c1e1f07513}, !- Handle
+  {e3f95af1-28af-4a74-9cee-a354ec0ceb1f}, !- Handle
   TARP;                                   !- Algorithm
 
 OS:ZoneCapacitanceMultiplier:ResearchSpecial,
-  {3b4980bb-5881-4958-b6b0-13874ff59072}, !- Handle
+  {ea28a287-7909-43f0-bbd1-b9221ccb3b9e}, !- Handle
   ,                                       !- Temperature Capacity Multiplier
```

Comment:
Seems like this field should get a 3.6 (since this particular test osm has ResidentialSimulationControls in its workflow). Not sure why it remains set to its default value...

Reply:
@joseph-robertson Thanks, addressed in SHA 0569578.